index.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Mobile Website</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="toolbar">
+      <nav class="nav-bar">
+        <a href="#"><img src="./images/logo.png" alt="logo" /></a>
+        <ul class="sid-cons">
+          <li class="p">Portoflio</li>
+          <li class="a">About</li>
+          <li class="c">Contact</li>
+          <li class="material-symbols-outlined">menu</li>
+        </ul>
+      </nav>
+    <div class="headline-container">
+      <div class="items1">
+        <h1 class="items item1">I'm Gasangwa Glad to see you !</h1>
+        <p class="items item3">
+          I am a software developer. I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need to be coded, don't hesitate to
+          contact me.
+        </p>
+        <div>
+          <h4 class="contact">LET'S CONNECT</h4>
```

Comment:
- [ ] You did a great job on the styles but for this project, you are supposed to stick to the Figma design as much as possible. Please fix the background images for the headline section and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166200154-13966082-9fb6-435e-88ab-23d6211f1e79.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166200190-8fd019b5-9c11-4a70-a859-ef0b9075697c.png)